extensions/adobe/experience/analytics/context-data.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/analytics/context-data",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "title": "Context Data",
+  "type": "object",
+  "description": "A free form field where Analytics context data will be captured.",
+  "definitions": {
+    "context-data": {
+      "properties": {
+        "https://ns.adobe.com/experience/analytics/contextData": {
+          "title": "Context Data Map",
```

Comment:
I would just call the field "Context Data" to match what people know it as.

Reply:
Will do